how-to/customize-workspace/client/src/framework/buttons.ts

```diff
@@ -84,3 +82,48 @@ export async function updateToolbarButtons(
 
 	return buttons;
 }
+
+export async function updateButtonColorScheme(): Promise<void> {
+	const platform = getCurrentSync();
+
+	const browserWindows = await platform.Browser.getAllWindows();
+
+	for (const browserWindow of browserWindows) {
+		await updateBrowserWindowButtonsColorScheme(browserWindow);
+	}
+}
+
+export async function updateBrowserWindowButtonsColorScheme(
+	browserWindow: BrowserWindowModule
+): Promise<void> {
+	const options = await browserWindow.openfinWindow.getOptions();
```

Comment:
This feels like it should be part of the button definition with v10 sdk. i.e. provide two sets of icons and the workspace code does this work. We can catch up tomorrow on whether we should propose it.

Reply:
does that break backwards compatibility? Do we need to make a note of that?

Reply:
I will add to changelog

Reply:
Hmm this might break sales studio as well so we might need v10 logic

Reply:
I didn't see anything themed icon paths in Sales Studio